### PR TITLE
Add the checkpoint of fresh boot

### PR DESCRIPTION
**Pull request by iccaszhulili**
There are two fix in this commit
1) The first one is add the checkpoint of fresh boot by using the
command "uptime"
2) fix the minor log changes of libvirt-guests for the negative
case.

Signed-off-by: Lily Zhu <lizhu@redhat.com>

### CODE REVIEW

**Comment by iccaszhulili** [comment]
# avocado run --vt-type libvirt managedsave.libvirt_guests
JOB ID     : 58b4ed105461e6b9c43402d4c126a42232aca6ea
JOB LOG    : /root/avocado/job-results/job-2019-01-02T14.07-58b4ed1/job.log
 (1/7) type_specific.io-github-autotest-libvirt.managedsave.libvirt_guests.positive_test.no_parallel_shutdown.suspend_on_shutdown.start_on_boot: PASS (33.65 s)
 (2/7) type_specific.io-github-autotest-libvirt.managedsave.libvirt_guests.positive_test.no_parallel_shutdown.suspend_on_shutdown.ignore_on_boot: PASS (34.70 s)
 (3/7) type_specific.io-github-autotest-libvirt.managedsave.libvirt_guests.positive_test.no_parallel_shutdown.shutdown_on_shutdown.start_on_boot: PASS (44.65 s)
 (4/7) type_specific.io-github-autotest-libvirt.managedsave.libvirt_guests.positive_test.no_parallel_shutdown.shutdown_on_shutdown.ignore_on_boot: PASS (28.75 s)
 (5/7) type_specific.io-github-autotest-libvirt.managedsave.libvirt_guests.positive_test.parallel_shutdown.shutdown_on_shutdown.start_on_boot: PASS (72.69 s)
 (6/7) type_specific.io-github-autotest-libvirt.managedsave.libvirt_guests.positive_test.parallel_shutdown.shutdown_on_shutdown.ignore_on_boot: PASS (53.26 s)
 (7/7) type_specific.io-github-autotest-libvirt.managedsave.libvirt_guests.negative_test.no_parallel_shutdown: PASS (38.13 s)
RESULTS    : PASS 7 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 309.32 s

File: libvirt/tests/src/conf_file/sysconfig_libvirt_guests/libvirt_guests.py [hunk 1]
```diff
@@ -179,6 +187,15 @@ def chk_save_files(status_error, on_shutdown, on_boot):
                                       "and been ignored on host boot, there "
                                       "should be save files for the guests.")
 
+    def boot_time():
+        booting_time = []
+        for vm in vms:
+            session = vm.wait_for_login()
+            status, time = session.cmd_status_output("uptime --since")
```
Comment (chunfuwen) [inline libvirt/tests/src/conf_file/sysconfig_libvirt_guests/libvirt_guests.py]: status can be _ if status never used

File: libvirt/tests/src/conf_file/sysconfig_libvirt_guests/libvirt_guests.py [hunk 2]
```diff
@@ -48,6 +49,13 @@ def chk_on_shutdown(status_error, on_shutdown, parallel_shutdown, output):
                                   be shutdown concurrently on shutdown
         :param output: apppended message from /var/log/messages
         """
+        if on_shutdown == "shutdown" and on_boot == "start":
+            second_boot_time = boot_time()
+            logging.debug("The second boot time is %s", second_boot_time)
+            if not all([i < j for i, j in zip(first_boot_time, second_boot_time)]):
```
Comment (chloerh) [inline libvirt/tests/src/conf_file/sysconfig_libvirt_guests/libvirt_guests.py]: if any([i >= j for i, j in zip(first_boot_time, second_boot_time)]):
More clear?

Comment (iccaszhulili) [reply libvirt/tests/src/conf_file/sysconfig_libvirt_guests/libvirt_guests.py]: updated

**Pull request merged**